src/sql/workbench/contrib/jobManagement/browser/jobsView.component.ts

```diff
@@ -561,10 +561,8 @@ export class JobsViewComponent extends JobManagementView implements OnInit, OnDe
 
 	private checkPreviousFilters(item): boolean {
 		for (let column in this.filterValueMap) {
-			if (column !== 'start' && this.filterValueMap[column][0].length > 0) {
-				let temp = this.filterValueMap[column][0] as unknown;
-				let arr = temp as [];
-				if (!arr.find(x => x === item[JobManagementUtilities.convertColNameToField(column)])) {
+			if (column !== 'start' && this.filterValueMap[column][0]) {
```

Comment:
@abist Could you look at this code and help me understand what the heck this is supposed to be doing? This check seems completely broken even without the find logic issue : 

1. filterValueMap seems to have two types of items added to it - the [actual filtered items](https://github.com/microsoft/azuredatastudio/blob/0846ec5e23fb0e2716baafbbc548c911c2409f0e/src/sql/workbench/contrib/jobManagement/browser/jobsView.component.ts#L335) but also an array of the [actual selected filter values for each column](https://github.com/microsoft/azuredatastudio/blob/0846ec5e23fb0e2716baafbbc548c911c2409f0e/src/sql/workbench/contrib/jobManagement/browser/jobsView.component.ts#L320)
1. What is this even supposed to be doing? I'm not understanding what "checkPreviousFilters" is supposed to mean - I think it's supposed to just check whether the item should be ignored (filtered out) or not?

What I have here is definitely not correct but I need more context on what all is happening here to figure out what is supposed to be happening

Reply:
@Charles-Gagnon It's been a while since I worked on this, but I'll pull and see what the issue is